app/components/form-field.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember';
+import FormField from 'ember-form-for/components/form-field';
+
+const {get, getWithDefault, set} = Ember;
+
+export default FormField.extend({
```

Comment:
We've had trouble with this update-on-focus-out workaround with checkboxes in Safari/Firefox. I didn't dig in too deeply but ended up simply overriding the behaviour so that checkboxes always update the underlying value instantly:

```javascript
    didInsertElement() {
        this._super(...arguments);

        // checkboxes don't work with `rawValue` in Safari/Firefox and shouldn't
        // have on-focus-out validation attached to them anyway
        if (get(this, 'control') === 'one-way-checkbox') {
            set(this, 'updateOnFocusOut', false);
        }
    },
```